tensorboard/examples/plugins/example_angular/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+.DS_Store
```

Comment:
Files specific to particular environments not generated by the project
itself, such as `.DS_Store`, should be in a user-level
`.gitignore_global` file rather than the project’s `.gitignore`.

See: <https://github.com/tensorflow/tensorboard/pull/558#issuecomment-332020237>

Reply:
Addressed in #2649